### DISCUSSION TITLE
Posibility to change the fixed wing cruise throttle setting in flight

### DISCUSSION THREAD

**Discussion by helicocrasher** [iNavFlight/inav#9444] [Ideas]
I love iNav on my fixed wings and often use cruise mode.
I some times would love to change the flying speed setting that is very much determined by the cruise throttle.
Fly slow and economical or fly fast to follow a friends plane. 
On the ground I can do this by selecting different battery profiles. 
In the air i can use throttle override but that is not convincing.

A global function to set the cuise throttle value, would be a nice option for my objectives..

Ony other solutions ??

**Comment by MrD-RC** [comment]
That is what the **Allow manual throttle override** setting is for on the advanced tuning page.

**Reply by helicocrasher** [reply]
Thanks Darren,
Yes I have manual throttle override set und use it sometimes, but it does not allow reducing and  it is not really easy to fine adjust.

**Reply by MrD-RC** [reply]
It shouldn’t need reducing. The cruise throttle should be optimum. Then use manual increase to increase it if you want to. 

The other option is to use Course Hold and have full manual control of the throttle. 

BTW, slow and economical is a not always the best option. It depends on your goals. Slow will give more flight time, but less distance. A slightly faster cruise (Carson speed) will give more distance but less flight time.

**Reply by helicocrasher** [reply]
Thanks Darren for your reply. I agree the slowest cruise speed is not necesarily providing the longest range  .. L/D max speed is in still air.
(And I had a read about carson speed ="L/D max +32%" interesting did not know this before).
Course hold is cool too but does not hold the altitude.  
I still dream of a global function for changing the cruise throttle :-)  but if thats not available ... no big issue.

**Reply by jongell** [reply]
Hi heliocrasher you could use in flight adjustments in air and some nice mixing on your radio to adjust the cruise throttle seemless in air. 

I have it setup up on my endurance plane. On one test I tried to reach maximum range efficiency hence I had to keep the plane in the optimal speed during the whole flight.

Take a look at this adjustments diff for INAV 6:

<img width="1440" alt="Screenshot 2024-01-02 at 11 39 48" src="https://github.com/iNavFlight/inav/assets/38889587/cbb52f4e-bbad-449d-a77e-c6211cd3a446">

# Adjustments [adjrange]
adjrange 0 0 3 1900 2100 32 5
adjrange 1 0 4 1900 2100 58 5

WIth opentx you can assign a switch to a channel to activate adjustment, then assign a trim switch to another channel for adjustment (high or low) and then the adjustment works in steps of 10us. 1% per click.

also in your OSD it is nice to add the cruise throttle so when adjusting you can see it blinking

**Reply by helicocrasher** [reply]
Thank you @jonasbouchraiet 
This looks great. Never used iNav Adjustments yet.
